src/App.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 
+type State = {
+  pressedKey: boolean,
+  userPressed: string,
+};
+
+export class Click extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: false,
+    userPressed: '',
```

Comment:
What the reason to have two of those values in state? You need only one actually